app/routes/task_routes.py

```diff
@@ -0,0 +1,115 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from ..models.task import Task
+import datetime
+import os
+import requests
+
+bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response({"message":f"{cls.__name__} {model_id} invalid"}, 400))
+
+    model = cls.query.get(model_id)
+    if not model:
+        abort(make_response({"message":f"{cls.__name__} {model_id} not found"}, 404))
+
+    return model
+
+def validate_task_dict(request_body):
```

Comment:
A better name might be `validate_task_request_body`! Also, the `request_body.get` calls on line 24 have explicit default values of `False` but since the `.get` default is `None`, and since `None` is falsy, you don't need to set them to `False` explicitly.